dealers_choice.html

```diff
@@ -1,15 +1,104 @@
 <!DOCTYPE html>
+<html>
   <head>
     <title>Polyamory</title>
+    <style>
+      body {
+        font-family: calibri;
+        margin: 0; 
+      }
+      /* * {
+        box-sizing: border-box;
+      } */
+      body > div > div {
+        float: left;
+        padding: 12px;
+        width: 28%;
+      }
+     
+      h1 {
+        color: blue;
+        font-family: Helvetica;
+        text-align: center;
+        text-decoration: underline;
+      }
+      h2, h3 {
+        color: blueviolet;
+        text-transform: upperCase;
+      }
+      .textSections {
+        background-color: lightgray;
+        color: black;
+        font-size: 14;
+        border: 1px solid blue;
+        padding: 2rem;
+      }
+      .where {
+        font-size: 10;
+        font-style: italic;
+      }
+      /* img {
+        max-width = 20px;
+        height: auto;
+      } */
+      .nav {
+        list-style-type: none;
+        /* for site index? */
+      }
+      .links {
+        text-size: 12;
+      }
+    </style>
   </head>
   <body>
     <h1>Polyamory</h1>
-    <h2>What is polyamory?</h2>
-    <p>Polyamory is the practice of, or desire for, intimate relationships with more than one partner, with the informed consent of all partners involved.[1][2] People who identify as polyamorous may believe in open relationships with a conscious management of jealousy and reject the view that sexual and relational exclusivity are necessary for deep, committed, long-term, loving relationships.[3][4] Others prefer to restrict their sexual activity to only members of the group, a closed polyamorous relationship which is usually referred to as polyfidelity.</p>
-    <p>Source: Wikipedia</p>
-    <h3>Is "polyamory" the same as ENM?</h3>
-    <p>ENM, or Ethical Non-Monogamy is is the practice of non-monogamous sexual relations that is distinguished from infidelity by the knowledge and consent of those involved, and from polygamy by the various partners not being in a single marriage. Forms of consensual non-monogamy include swinging, polyamory, open relationships and cuckolding fetishism.
-        Polyamory is just one form of ENM. 
-    </p>
+    <div>
+    <div class ="textSections">
+      <h2>What is polyamory?</h2>
+      <p>
+        Polyamory is the practice of, or desire for, intimate relationships with
+        more than one partner, with the informed consent of all partners
+        involved. 
+        <br><br>People who identify as polyamorous may believe in open
```

Comment:
`<br>` (and `<strong>` and `<em>`) shouldnt be used in html- it's because with br, there's a LOT of repeating code and we like to limit repeating code 
it's because- what if in the future, you want to increase the spacing between some of the elements on the page?
we'd have to add many `<br>` everywhereeee (´・◠・｀)  

it's much better to just use css for this purpose- with margins and padding